File: test/fixtures/ocaml/get_with_data.ml
open Lwt
open Cohttp
open Cohttp_lwt_unix

let uri = Uri.of_string "http://localhost:28139/synthetics/api/v3/monitors?test=2&limit=100&w=4" in
let headers = Header.init_with "X-Api-Key" "123456789" in
Client.get ~headers uri
>>= fun (resp, body) ->
  (* Do stuff with the result *)

File: test/fixtures/julia/get_with_data.jl
using HTTP

query = [
    "test" => "2",
    "limit" => "100",
    "w" => "4"
]

headers = Dict(
    "X-Api-Key" => "123456789"
)

resp = HTTP.get(
    "http://localhost:28139/synthetics/api/v3/monitors",
    headers,
    query=query,
    verbose=1
)
